charts/filsnap/templates/nodelocker/deployment.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "filsnap.fullname" . }}-nodelocker
+  labels:
+    {{- include "filsnap.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "filsnap.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.nodelocker.annotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "filsnap.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "filsnap.nodelocker.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}-nodelocker
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.nodelocker.image.repository }}:{{ .Values.nodelocker.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.nodelocker.image.pullPolicy }}
+          command:
+            - filsnap
+          args:
+            - nodelocker
+            - run
+          env:
+          - name: FILSNAP_LOG_LEVEL
+            value: {{ .Values.nodelocker.logging.level | quote }}
+          {{- if .Values.nodelocker.logging.named }}
+          - name: FILSNAP_LOG_LEVEL_NAMED
+            value: {{ include "filsnap.logLevelNamed" . | quote }}
+          {{- end }}
+          - name: FILSNAP_NODELOCKER_SERVICE_LISTEN
+            value: "0.0.0.0:5100"
+          - name: FILSNAP_NODELOCKER_OPERATOR_LISTEN
+            value: "0.0.0.0:5101"
+          {{- with .Values.nodelocker.additionalEnv }}
+          {{- toYaml . | nindent 10 }}
+          {{- end }}
+          ports:
+          - containerPort: 5100
+            name: service
+            protocol: TCP
+          - containerPort: 5101
+            name: operator
+            protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /liveness
+              port: operator
+          readinessProbe:
+            httpGet:
+              path: /readiness
+              port: operator
```

Comment:
https://github.com/travisperson/filsnap/blob/main/pkg/nodelocker/service/service.go#L78-L90

Reply:
Not sure if we want to add `initialDelaySeconds` or `periodSeconds`

Reply:
It would depend if the default is long enough

Reply:
https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes

The service starts up very fast, so I think we can run with the defaults.